packages/app/Dockerfile

```diff
@@ -9,14 +9,13 @@ WORKDIR /usr/src/app
 COPY package*.json ./
 RUN npm install
 COPY . .
-RUN npm run build
 
 EXPOSE 3000
 EXPOSE 4000
 
 CMD if [ "$NODE_ENV" = "production" ]; \
-	then \
-	npm run start; \
-	else \
-	npm run dev; \
-	fi
+  then \
+  npm run build && npm run start; \
```

Comment:
We shouldn't be running builds in production, that's not a good practice. Building at runtime leads to a slower startup and risk of the build failing on service restart.

I'm not sure why this .next issue just started cropping up, as this Dockerfile has been in place for years, but regardless, we need to fix it. I'm not going to hold this PR up though, opened an issue to handle this, but we definitely should explore this and revert back to moving the build to the Docker build step: https://github.com/nasa/gesdisc-meditor/issues/53

Reply:
That makes two of us (who aren't sure why this just started happening).
1. Did the issue affect you, too?
2. I'm not sure what you mean by "running builds in production". Could you explain this at some point?

Either way, I'll go ahead and merge, because this seems an easy thing to revert.